apple/internal/aspects/resource_aspect.bzl

```diff
@@ -131,7 +131,18 @@ def _apple_resource_aspect_impl(target, ctx):
     # The name of the bundle directory to place resources within, if required.
     bundle_name = None
 
-    if ctx.rule.kind == "objc_library":
+    if ctx.rule.kind == "cc_library":
```

Comment:
nit: can we list the `cc_*` rules last, since they will be less common, and thus can be in the slow path?

Reply:
updated